rendering: true
dynamics:
  substeps: 1
  kinematic_simulation: true

solver:
  rollouts: 200
  lambda: 0.0001 #0.001
  h: 10.0
  step_size: 0.01
  horizon: 1.0

  cost_ratio: 0.2
  caching_factor: 0.3
  filtering: true
  substeps: 1

  filter_type: 1
  filter_order: 2
  filter_window: 50

  adaptive_sampling: false
  input_variance: [0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15]

  discount_factor: 1.0
  verbose: false
  threads: 1
  debug_print: false

initial_configuration: [0.0, -0.52, 0.0, -1.785, 0.0, 1.10, 0.69]
linear_weight: 100.0
angular_weight: 10.0
obstacle_radius: 0.15

policy_update_rate: 0.0  # execute as fast as possible
reference_update_rate: 10.0
publish_ros: true
ros_publish_rate: 10.0

static_optimization: false
sim_dt: 0.01